tests/test_integration.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import flexmock
+import pytest
+
+from apigentools.commands.init import InitCommand
+
+def test_init(tmpdir):
+    original_dir = os.getcwd()
+    try:
+        temp_dir = tmpdir.mkdir("test_init_git_dir")
+        os.chdir(temp_dir)
+        args = flexmock.flexmock(no_git_repo=False, projectdir=temp_dir)
+        cmd_instance = InitCommand({}, args)
+        cmd_instance.run()
+        #sets do not presume the output of os.walk() will be ordered
+        dir_entries = set(dir_entry[0] for dir_entry in os.walk("."))
```

Comment:
can we just pass temp_dir into the os.walk call and avoid the need for the overall try/finally?

Reply:
Same for the other trys here :)

Reply:
:galaxy-brain:

Reply:
:galaxy_brain: